modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/list-item-with-icon/index.js

```diff
@@ -0,0 +1,22 @@
+const plusIcon = fragmentElement.querySelector('#plus-icon');
+const minusIcon = fragmentElement.querySelector('#minus-icon');
+const customIcons = fragmentElement.getElementsByClassName('custom-icon');
+const collapseDiv = fragmentElement.querySelector('#collapseDiv');
+
+for (let i = 0; i < customIcons.length; i++) {
+	const icon = customIcons[i];
+
+	icon.onclick = function () {
+		let flag = collapseDiv.classList.contains('collapse');
+		if (flag) {
```

Comment:
Enter between 10 and 11

```suggestion

		if (flag) {
```